openlibrary/templates/lists/widget.html

```diff
@@ -172,7 +173,7 @@
                 <div class="reading-lists">
                   <p class="reading-list-title">$_('My Reading Lists:')</p>
                   <div class="my-lists">
-                    $for i, list in enumerate(lists):
+                    $for i, list in enumerate(reversed(sorted(lists, key=lambda x:x.last_modified))):
```

Comment:
this can be simplified using built-in functionality. You'll need the following import:

`from operator import attrgetter`

```suggestion
                    $for i, list in enumerate(sorted(lists, key=attrgetter('last_modified'), reverse=True)):
```

Reply:
Ignore this suggestion. Imports aren't allowed in templates!